high_card.rb

```diff
@@ -1,5 +1,44 @@
+require 'pry'
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "Jack", "Queen", "King", "Ace"]
+suits = [ "Hearts", "Spades", "Clubs", "Diamonds" ]
+deck = []
+player_input = ""
 players = []
+
+# Creates Deck with 52 cards and index values for comparing
+ranks.each_with_index do |rank, i|
+  suits.each do |suit|
+    deck << [i + 2,rank,suit]
+  end
+end
+
+# Shuffles Deck
+deck = deck.shuffle
+# puts deck
+
+# Get names of players and push them to array of players
+while player_input != "play" do
+  puts "#{players.length} players so far. Enter a player name, or type 'play'."
+  player_input = gets.chomp.to_s
+  players << player_input
+end
+players.pop() #Removes 'play' as a player
```

Comment:
You don't have to do player.pop() if you change your code around a little, for example: 
```
 while player_input do
   puts "#{players.length} players so far. Enter a player name, or type 'play'."
   player_input = gets.chomp.to_s
   break if player_input == 'play'
   players << player_input
 end
```